addons/binding/org.openhab.binding.rotelra1x/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,23 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: RotelRa1x Binding
+Bundle-SymbolicName: org.openhab.binding.rotelra1x;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ org.openhab.binding.rotelra1x,
+ org.openhab.binding.rotelra1x.handler,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
```

Comment:
This binding is not yet fully up-to-date with the rest of the openhab/smarthome ecosystem, given that the import package `org.eclipse.jdt.annotation;resolution:=optional` is missing. You might want to update